dashboard/src/recoil/selectors.js

```diff
@@ -134,6 +154,35 @@ export const itemsGroupedByPersonSelector = selector({
         outOfActiveList: latestTeamFilteringItem.outOfActiveList,
         def: "created",
       });
+      if (person.history?.length) {
+        // history is sorted by date from the oldest to the newest (ascending order)
+        // we want to loop it from the newest to the oldest (descending order)
+        for (let i = person.history.length - 1; i >= 0; i--) {
+          const historyEntry = person.history[i];
+          personsObject[person._id].interactions.push(historyEntry.date);
+          if (historyEntry.data.assignedTeams) {
+            const currentTeams = historyEntry.data.assignedTeams.newValue || allTeamIds;
+            const previousTeams = historyEntry.data.assignedTeams.oldValue || allTeamIds;
+            const newlyAddedTeams = currentTeams.filter((t) => !previousTeams.includes(t));
+            const removedTeams = previousTeams.filter((t) => !currentTeams.includes(t));
+            for (const teamId of newlyAddedTeams) {
+              assignedTeamsPeriods[teamId] = assignedTeamsPeriods[teamId].map((period) => {
```

Comment:
J'ai un crash à cause de cette ligne (page blanche direct) quand je vais sur les stats

```suggestion
              assignedTeamsPeriods[teamId] = (assignedTeamsPeriods[teamId] || []).map((period) => {
```